client/app/visualizations/components/visualizationsSettings.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { extend } from "lodash";
+import Icon from "antd/lib/icon";
+import Tooltip from "antd/lib/tooltip";
+
+export const visualizationsSettings = {
+  NumberFormatSpecs: (
+    <Tooltip
+      title={
+        <React.Fragment>
+          Formatting Numbers
+          <i className="fa fa-external-link m-l-5" />
+        </React.Fragment>
+      }>
+      <a
+        className="visualization-editor-context-help"
+        href="https://redash.io/help/user-guide/visualizations/formatting-numbers"
+        target="_blank"
+        rel="noopener noreferrer">
+        <Icon className="m-l-5 m-r-5" type="question-circle" theme="filled" />
+      </a>
+    </Tooltip>
```

Comment:
> This probably means we need to change the API of HelpTrigger to take URL and text instead of constants like it does now, but I think it's a reasonable tradeoff.

I though about it too, to do that we also need to add a check to make sure the base url is from `redash.io`.

What made me use this option instead is because that's the only usage we have for it now (and I wanted to focus on the settings part first, rather than changing the HelpTrigger). I was actually considering giving up on the HelpTrigger as it was just one use, but on a second thought it's very likely that we add more entries in the future, so I'll just change it to also accept a separate link and text.

Reply:
HelpTrigger updated to receive `href` and `title` props separately as an alternative to constants. It also accepts URLs ouside of `redash.io` now, but renders a regular link in such case.